safe/utils_test.go

```diff
@@ -14,19 +14,19 @@ func init() {
 
 const (
 	// testSafeContent has one account in it, "foo".
-	testSafeContent = `-----BEGIN PGP SIGNATURE-----
-
-wx4EBwMIEp9YTGqqetBgwXj5+80vLdBMeIvLS49/GPDS4AHkVDeGlKy/bXalpl9M
-q7YwLOEBo+DO4CTh9ujgVeKriVie4HjmoykOiH3REZLAJO7U3ejMIP4onlD0u6SJ
-vD4U2ipsQIWhWaWfASmpl6T0Qq39tDqE8XZjFcTt/Btujfb5zoPpbeA84s53sEXg
-suDC4K7k5IYOU66CG3XbOaafkrrfmeKCjJGG4aZyAA==
-=Wzqv
------END PGP SIGNATURE-----`
+	testSafeContent = `-----BEGIN PGP MESSAGE-----
```

Comment:
Note- We should be able to drop in the `MockBackend` here and simplify these tests once this and https://github.com/bndw/pick/pull/65 are merged.